cmake/compiler/gcc/target_warnings.cmake

```diff
@@ -108,3 +108,10 @@ endmacro()
 macro(toolchain_cc_cpp_warning_error_misra_sane dest_var_name)
   set_ifndef(${dest_var_name} "-Werror=vla")
 endmacro()
+
+# List the warnings that are not supported for C++ compilations
+
+set(CXX_EXCLUDED_OPTIONS
```

Comment:
list(APPEND

then it could be populated elsewhere as well.

Reply:
thanks; updated.